src/components/PressableWithDelayToggle.js

```diff
@@ -98,9 +98,10 @@ function PressableWithDelayToggle(props) {
                 <>
                     <Text
                         suppressHighlighting
-                        style={[styles.mr1, ...props.textStyles]}
+                        style={props.textStyles}
                     >
                         {props.isDelayButtonStateComplete && props.textChecked ? props.textChecked : props.text}
+                        &nbsp;
```

Comment:
mr1 in this context doesn't work. That's why I replaced it with space

Reply:
Yep, I think it's fine to make those separate issues but please note it in the PR so that our QA team doesn't mark them as regressions from this PR. Once this is live, they should be reported as new bugs.

Reply:
PR updated